overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -75,19 +75,28 @@ type command interface {
 	Execute(args []string) error
 }
 
+type argDesc struct {
+	name string
+	desc string
+}
+
 type commandInfo struct {
 	shortHelp string
 	longHelp  string
 	generator func() command
+	optDescs  map[string]string
+	argDescs  []argDesc
```

Comment:
Ah, thanks for spotting. I've initially followed the pattern found in main snap command, but later abandonded it and forgot about this leftover. Removed.